packages/client/src/ontology/StandardOntologyProvider.ts

```diff
@@ -65,7 +67,17 @@ export const createStandardOntologyProviderFactory: (
       return loadInterfaceDefinition(client, key);
     }
 
-    function makeGetter<N extends ObjectOrInterfaceDefinition>(
+    async function loadQuery(
```

Comment:
Little confused what this is for. So is this equivalent to the fetchMetadata stuff we'll have for objects?